features/pause_and_resume.feature

```diff
@@ -54,55 +54,6 @@ Feature: Pause and Resume User Journey
     When they give their consent
     Then they should be successfully verified
 
-  Scenario: User pauses and resumes from an IDP e-mail link, on same device, and resumes registration
```

Comment:
Did you mean to move these to another feature or are you just removing them?

Reply:
added a comment just now ;)

Reply:
No, thats OK ... I see your point!